ssl/tls13_enc.c

```diff
@@ -469,7 +471,6 @@ static int derive_secret_key_and_iv(SSL *s, int sending, const EVP_MD *md,
 
     return 1;
  err:
```

Comment:
Might as well remove all the "goto err" and just return 0.

Reply:
fixed